src/components/lyne-tab-group/lyne-tab-group.tsx

```diff
@@ -0,0 +1,294 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Method,
+  Prop
+} from '@stencil/core';
+import { InterfaceLyneTabGroupTab } from './lyne-tab-group.custom';
+import throttle from '../../global/helpers/throttle';
+
+/**
+ * @slot tab-bar - Pass an heading tag to display a label in the tab bar.
+ * Example: `<h1>Tab label</h1>`
+ * @slot unnamed - Pass html-content to show as the content of the tab.
+ * Wrap the content in a div:
+ * This is correct: `<div>Some text <p>Some other text</p></div>`
+ * This is not correct: `<div>Some text</div><p>Some other text</p>`
+ */
+
+const tabObserverConfig: MutationObserverInit = {
+  attributeFilter: [
+    'active',
+    'disabled'
+  ]
+};
+
+@Component({
+  shadow: true,
+  styleUrls: {
+    default: 'styles/lyne-tab-group.default.scss',
+    shared: 'styles/lyne-tab-group.shared.scss'
+  },
+  tag: 'lyne-tab-group'
+})
+
+export class LyneTabGroup {
+
+  /**
+   * Sets the initial tab. If it matches a disabled tab or exceeds the length of
+   * the tab group, the first enabled tab will be selected.
+   */
+  @Prop() public selectedIndex = 0;
+
+  /**
+   * On selected tab change
+   */
+  @Event({
+    eventName: 'lyne-tab-group_tab-change'
```

Comment:
lyne-tab-group_tab-change or lyne-tab-group_tab-changed? I added this topic to our developer meeting list.